views/shelf.erb

```diff
@@ -68,6 +71,9 @@ Renew all could be too slow. Maybe reconsider.
           <span>&#183; <%= loan.author %></span>
         </td>
         <td>
+          <% if loan.message? %>
+            <span class="action-<%=loan.message_status%>"><%= loan.message%></span>
+          <% end %>
```

Comment:
The inline messages do not look good. Halp.

Reply:
Sure thing! I'll get to this sometime this week, hopefully, tomorrow. Thanks, @niquerio!

Reply:
I saw the already existing but unused `./css/_tags.scss` styles. I updated the `span` class and `tags--error` class name to `tags--fail`. I wasn't sure if it was better to change the class name or to change all mentions of `:fail` to `:error`.